packages/cli/test/Prompt.test.ts

```diff
@@ -0,0 +1,49 @@
+import * as Prompt from "@effect/cli/Prompt"
+import * as MockTerminal from "@effect/cli/test/services/MockTerminal"
+import type { Terminal } from "@effect/platform"
+import { Effect } from "effect"
```

Comment:
To keep things consistent:

```suggestion
import * as Effect from "effect/Effect"
```